src/app/projects/components/project-settings/project-settings.component.html

```diff
@@ -214,18 +214,15 @@
             </div>
         </ng-template>
     </div>
-    <div *ngIf="settingModals.projectExport.projectSize" class="mt-4 flex flex-row justify-between "
+    <div *ngIf="settingModals.projectExport.projectSize" class="mt-4 justify-between"
```

Comment:
i think jusitfy-between needs a flex or something like that so we can maybe remove the class?

Reply:
yes, that's true, the class does not have any impact on the element since it does not have the flex display, removed :D